lib/rust/prelude/src/lib.rs

```diff
@@ -483,22 +483,34 @@ pub trait CellSetter: HasItem {
 
 /// Generalization of modify utilities for structures similar to [`Cell`].
 pub trait CellProperty: CellGetter + CellSetter + ItemClone {
-    /// Updates the contained value using a function and returns the new value.
+    /// Update the contained value using the provided function and return the new value.
     fn update<F>(&self, f: F) -> Self::Item
     where F: FnOnce(Self::Item) -> Self::Item {
         let new_val = f(self.get());
         self.set(new_val.clone());
         new_val
     }
 
-    /// Modifies the contained value using a function and returns the new value.
+    /// Modify the contained value using the provided function and return the new value.
     fn modify<F>(&self, f: F) -> Self::Item
     where F: FnOnce(&mut Self::Item) {
         let mut new_val = self.get();
         f(&mut new_val);
         self.set(new_val.clone());
         new_val
     }
+
+    /// Update the contained value using the provided function without returning the new value.
+    fn update_<F>(&self, f: F)
+    where F: FnOnce(Self::Item) -> Self::Item {
+        self.update(f);
+    }
+
+    /// Modify the contained value using the provided function without returning the new value.
+    fn modify_<F>(&self, f: F)
+    where F: FnOnce(&mut Self::Item) {
+        self.modify(f);
+    }
 }
```

Comment:
Why we need this? We can easily discard returning value anywhere we want using `{ ... ; }`

Reply:
As agreed via phone, let's leave it. The issue is that rust autoformatter formats code like this `foo.map(|t| { t.modify(f); })` in 2 lines and it is not configurable.